tests/files/rbdigital/response_catalog_delta.json

```diff
@@ -20,15 +20,15 @@
         "images": [
           {
             "name": "small",
-            "url": "http://images.oneclickdigital.com/78226/78226_image_71x108.jpg"
+            "url": "http://images.rbdigital.com/78226/78226_image_71x108.jpg"
```

Comment:
Same with these images, and anything that currently matches "images.rbdigital.com".